substrate/frame/revive/mock-network/src/lib.rs

```diff
@@ -19,7 +19,7 @@ pub mod parachain;
 pub mod primitives;
 pub mod relay_chain;
 
-#[cfg(all(test, feature = "riscv"))]
+#[cfg(all(test, feature = "FAIL-CI @pg"))]
```

Comment:
@pgherveou if i disable this gate, then there are compile errors in the tests file.  
Not sure if i am missing something, but there seems to be no `riscv` feature either so the code is just disabled.

Reply:
Good catch, looks like this is something we missed when we forked pallet-contracts
will fix it (tracking [here](https://github.com/paritytech/polkadot-sdk/issues/5683))

Reply:
fixed here https://github.com/paritytech/polkadot-sdk/pull/5684